src/app/containers/MostRead/Canonical/index.jsx

```diff
@@ -52,14 +65,39 @@ const CanonicalMostRead = ({ endpoint }) => {
   }, [endpoint, numberOfItems, datetimeLocale, lastUpdated, script, service]);
```

Comment:
Getting a warning here that `timezone` should be included

Reply:
ah yes, forgot to add timezone here previously 👍